src/model/digest.v1.yaml

```diff
@@ -0,0 +1,24 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Digest
+type: object
+allOf:
+  - $ref: ../snippets/digest.v1.yaml
+  - properties:
+        content:
```

Comment:
https://github.com/elifesciences/api-validator-python will come in handy for whatever implements this

Reply:
I didn't know this library provided the ability. I am playing with implementing some JSON output in the https://github.com/elifesciences/digest-parser project. Python 3 would be ok there, but if we want the elife-bot to run it, then the min Python 3.5 of api-validator-python might need some creative solutions. Maybe dockerise the conversion or something?